src/store/UserStore.tsx

```diff
@@ -19,17 +22,17 @@ class UserStore {
   }
 
   get jwtToken(): string | undefined {
-    return keycloak.token
+    return keycloak.token;
   }
 
   get xsrfToken(): string | undefined {
     return getCookie("XSRF-TOKEN");
   }
 
-  saveProfileToStorage(profile?: UserProfile) {
+  saveProfileToStorage(profile?: UserProfile | null) {
     if (profile) {
       this.profile = profile;
-      localStorage.setItem('user', JSON.stringify(this.profile));
+      localStorage.setItem("user", JSON.stringify(this.profile));
```

Comment:
Chyba powinno być "profile" zamiast "user": 
```suggestion
      localStorage.setItem("profile", JSON.stringify(this.profile));
```